test/CpmsClientTest/Bootstrap.php

```diff
@@ -95,11 +94,14 @@ protected function setPaths()
         );
 
         if (file_exists(static::$dir . "/autoload_classmap.php")) {
+            /** @psalm-suppress UnresolvableInclude */
             $classList = include static::$dir . "/autoload_classmap.php";
 
             spl_autoload_register(
-                function ($class) use ($classList, $basePath) {
+                function ($class) use ($classList) {
+                    /** @psalm-suppress UnresolvableInclude */
                     if (isset($classList[$class])) {
```

Comment:
Does something like:
```
array_key_exists('listeners', $config) &&
            is_array($config['listeners'])
```
would make suppression not needed?

Reply:
Not sure what you mean?

Reply:
Checked and it does not fix the need for suppression. Resolved